usama/sprint1/app.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+import os
+from aws_cdk import core as cdk
+from aws_cdk import core
+from infra.infra_stack import InfraStackUsama
```

Comment:
- core is imported twice, you can reduce it to 1

Reply:
That's crazy me who didn't notice. Well I have changed it now.